include/superior_mysqlpp/types/blob_data.hpp

```diff
@@ -17,15 +17,19 @@
 
 namespace SuperiorMySqlpp
 {
-    template<std::size_t N>
-    class BlobDataBase : public ArrayBase<N>
+    template<std::size_t N, bool TerminatingZero = false>
+    class BlobDataBase : public ArrayBase<N, TerminatingZero>
```

Comment:
Similarly as in comment above (ArrayBase). User can have blob data of e.g. type integer or some structure. This way they have to copy data via e.g. `memcpy` to aligned storage to ensure they don't have problems with aliasing.